src/packResourceProvider.cpp

```diff
@@ -67,6 +69,25 @@ std::vector<string> PackResourceProvider::findResources(const string searchPatte
 
 void PackResourceProvider::addPackResourcesForDirectory(const string directory)
 {
+#ifdef _WIN32
+    WIN32_FIND_DATAA data;
+    assert(directory.endswith("/"));
```

Comment:
Hench, this one...

Reply:
🤦 I thought both were in Serious Proton (and thought I dreamed the second one since I could no longer find it 😂 )

Reply:
Never said the code is well organized ;-)

Reply:
It's not *so* bad actually.